common/src/main/scala/com/avast/cactus/cactus.scala

```diff
@@ -1,8 +1,29 @@
 package com.avast
 
-import org.scalactic.Or
+import cats.data.NonEmptyList
+import cats.syntax.either._
+import org.scalactic.{Every, Or}
 
 package object cactus {
-  type CactusFailures = org.scalactic.Every[CactusFailure]
-  type ResultOrErrors[A] = A Or CactusFailures
+  type EveryCactusFailure = Every[CactusFailure]
```

Comment:
Shouldn't this be hidden or removed?

Reply:
It cannot be. These aliases are used in generated code thus they must have the same visibility as if the user write the code by himself.